### DISCUSSION TITLE
key "Principled BSDF" not found

### DISCUSSION THREAD

**Discussion by mokiduka** [carson-katri/dream-textures#561] [Q&A]
![A@H)ZL_N0GWB@KX@QM~5J(J](https://user-images.githubusercontent.com/108580478/217916225-0c7dcc35-536f-4e8f-ad15-97af3d444daf.png)
I do not know why this happens, have tried many ways still not solved, only installed this one model (depth)

**Comment by KrashlTest** [comment]
Hi! I had the same mistake. After I switched the Blender language to English, everything worked.
